extensions/net/README.md

```diff
@@ -0,0 +1,3 @@
+# deno_net
+
+This crate implements networking APIs.
```

Comment:
Ideally a bit more detail here. List the ops it provides?

Reply:
Added a list of ops